### ISSUE TITLE
There BEE troubles ahead ....

### ISSUE THREAD

**Issue by b0bst3r** [Beyond-Reality/Beyond-Reality-Divergence#212]
Firstly there's a lot of hives missing (forest/meadows/magic ones).

Secondly the temperatures of biomes make it impossible to get going, you really need a normal/normal biome and there just isn't any:

Biome Name      Temp        Humidity

Shrubland       Hot     Normal
Desert Oil Field        Hot         Arid
Tropical Shrubland      Hot     Normal
Woodland        Hot         Normal
Ender Forest        Warm        Damp
Ocean       Warm        Normal
Gravel Beach        Cold        Normal
Tundra      Cold        Normal
Magical Forest      Warm        Normal
Rainbow Forest      Hot     Damp
Nether      Hellish     Arid

**Comment by Dominance** [comment]
So quiet a few biomes missing from the list. But I am guessing they are no better. Might have to work on seasons a bit. or change forestry a bit.

**Comment by b0bst3r** [comment]
I couldn't find any of the others but Foo brought this up on he server after he tried to find a non-hot one, so I went looking.

**Comment by mrbaggins** [comment]
Yeah, I've really struggled to find hives, they are REALLY rare, and I've got more Rocky / chromatic ones than I do normal forestry ones.

**Comment by foopex** [comment]
I've managed to make it work, I've found 50 hives now by changing how/where I look.  I've found lots of tropicals in Woodlands and some in shrublands, and meadows spread around woodlands\ender forest\shrublands etc.  So far the hardest for me to find is forest bees.

Now, for apiaries, it seems that their function does change with the temporatures and/or seasons, I haven't found a pattern yet beyond a few days worth of beekeeping.  I'm looking at this as a challenge to overcome at this point with alvearies, we'll see.

**Comment by mrbaggins** [comment]
How long is a "season" or do they just sweep across the land semi-randomly? Because I have areas that have become snowy and then not, meanwhile my home location I've not noticed a change at all.

**Comment by b0bst3r** [comment]
Kinda changing the subject but to answer your question a season lasts 24 real hours.  If you gen new chunks whilst it's winter those new chunks will be covered in snow, once you leave the chunks that snow will stay until it's loaded again in a non wintery season.  My advice is it best to gen new chunks in the summer :)

**Comment by Dominance** [comment]
Working on a solution. Moving to 1.2.8

**Comment by foopex** [comment]
I've managed to make the imperials and industrials and start marking my alvearies and whatnot by hand without Genetics.  It  was hard with the narrow windows of opportunity between seasons, but ultimately possible.

**Comment by SkuxxDeluxe** [comment]
So i have more basic bee's than i know what do with.
Botania lets you access every basic bee type for mana, and along with the forest types, you get nether/end types too etc.

In terms of temperature, have you tried having the bee's inside a glass greenhouse (for the greenhouse effect)?

I havnt had a problem with bee's yet.

**Comment by tentonaxe** [comment]
@SkuxxDeluxe 
glass greenhouse doesn't appear to have any effect on the Temperature or Humidity when it comes to apiaries, unless i'm doing it wrong.

http://puu.sh/l6hZ5/83af1d9fc4.jpg same temp/humity inside as outside. http://puu.sh/l6i1K/8020d280b3.jpg

**Comment by mrbaggins** [comment]
Yeah, apiaries don't respond to anything like that I don't think.

**Comment by SkuxxDeluxe** [comment]
So, found a semi-easy solution.

Breed from rocky bees, they are normal/noraml and tolerate +/-2, which means pretty much every biome on your list. They have the added bonus of being both diurnal and nocturnal and cave dwelling.

Sure, you gotta breed that trait into all your other bees, but for 'most' of them, thats not a problem, and for the rest the alveary will do the trick.

they are also VERY easy to find :p

**Comment by SkuxxDeluxe** [comment]
Also, beaches are normal, normal

**Comment by tentonaxe** [comment]
mine are icy normal, but i'll give rocky a try, last time i tried in shrubland and it was too hot.

**Comment by b0bst3r** [comment]
All biome temperatures are controlled by seasons, by biome and I believe by time of day.  So yeah saying one is normal normal might only occur during a particular season and at a particular time of day.

**Comment by foopex** [comment]
@tentonaxe

We don't have actual beaches on our world, we have gravel beaches everywhere, and they are all icy.  Our world seed is terrible.

**Comment by SkuxxDeluxe** [comment]
@b0bst3r, thats true. Not sure about the day affecting temperatures (it didnt in the small testing i did with it), but there is some random fluctuations for sure.

 I guess what im sayin is, in regards to for your post where you say 'its impossible to get going', im saying it is possible.

The seasonal temperature changes can be a little brutal, but it just makes things slow and hard, not impossible. I live and breed in a Roofed Forest, which was warm/damp for winter, and is now hot/damp in spring.
I've already got a stable cultivated branch which works with this biome, even with the changed temp after about 8-10 hours of working on forestry.

I do think that perhaps plains should be normalised to 'normal/normal, in say spring and autumn. That might make it too easy though hehe.

Any other thoughts on what @Dominance should do about it?

**Comment by mrbaggins** [comment]
Bringing this up as a solution and another question:

Taiga for me was normal normal. I'm guessing that's because it's summer (i honestly don't know if that's true), so everything is a bit warmer than normal.

Is there anyway to tell what temperature / humidity places are? Or what season it is?

**Comment by tentonaxe** [comment]
Calendar for season, thermometer and hydration thingy for wetness

**Comment by Belgabor** [comment]
Another issue, since Forestry villagers are disabled, how do we get Monastic bees?

**Comment by mrbaggins** [comment]
Not sure. I've only just kicked off bees properly.

**Comment by Dominance** [comment]
Looks like forestry has a greenhouse function in the 1.9 branch so this will be fixed in divergence 2 at least.
